soda_machine.rb

```diff
@@ -7,12 +9,17 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.count
   end
 
   def find_soda(soda_brand)
+    @sodas.find { |soda| soda.brand == soda_brand }
   end
 
   def sell(soda_brand)
+    soda = find_soda(soda_brand)
+    return nil if soda == nil
```

Comment:
This is weird. why create a variable that stores nil? What's a different way to approach making sure there is a soda to sell?